Assets/Xbox Live/Scripts/UserProfile.cs

```diff
@@ -104,8 +162,9 @@ public IEnumerator SignInAsync()
             throw new Exception("Sign in Failed");
         }
```

Comment:
I think we should change this code.  look into [https://github.com/Microsoft/xbox-live-unity-plugin/issues/65](https://github.com/Microsoft/xbox-live-unity-plugin/issues/65)